language-plutus-core/src/Language/PlutusCore/Constant/Apply.hs

```diff
@@ -22,15 +22,14 @@ module Language.PlutusCore.Constant.Apply
     , runApplyBuiltinName
     ) where
 
+import           PlutusPrelude
+
 import           Language.PlutusCore.Constant.Dynamic.Instances ()
 import           Language.PlutusCore.Constant.Name
 import           Language.PlutusCore.Constant.Typed
```

Comment:
We should just have `Language.PlutusCore.Constant` that reexports all of those, but that's for later.

Reply:
We in fact to have it. I just didn't realize it's the `Language.PlutusCore.Constant.Apply` module and so it can't refer to `Language.PlutusCore.Constant` (which among other things reexports stuff from  `Language.PlutusCore.Constant.Apply`).